### DISCUSSION TITLE
Add result filename to output fields from "TextFileOutput"

### DISCUSSION THREAD

**Discussion by AlexLogov** [apache/hop#4741] [Ideas]
If we add  "filename" field in the output fields for transform  "TextFileOutput"  than it can be used  in pipeline. 

In this transform filename can be dynamic change (add timestamp or by spliting) , byt filename not visible into pipeline.  There is no way to find out the file name, save it, etc.  With save relation to current row.

May I suggest a pull request with this change?

**Comment by hansva** [comment]
Of course! we are always happy with PR's that fix or implement features.
We have implemented features so anyone can "claim" a ticket they are working on, in this case you could use the `.take-issue` command on #4697 to assign it to yourself.
